odoo/tests/common.py

```diff
@@ -358,6 +359,15 @@ def shortDescription(self):
     def assertItemsEqual(self, a, b, msg=None):
         self.assertCountEqual(a, b, msg=None)
 
+    @contextmanager
+    def patch_now(self, t):
+        patcher = freeze_time(t)
+
+        try:
+            patcher.apply()
+        finally:
+            patcher.revert()
```

Comment:
Incorrect: the method does not `yield`!

And why do you make it so complicated?  This just works, doesn't it?
```python
def patch_now(self, t):
    return freeze_time(t)
```

Reply:
whoops